testing/integration/tests/fil-events-actor/src/actor.rs

```diff
@@ -116,7 +116,9 @@ pub fn invoke(params: u32) -> u32 {
 
             if counter > 0 {
                 let params = fvm_ipld_encoding::to_vec(&counter).expect("failed to serialize");
-                sdk::send::send(&our_addr, EMIT_SUBCALLS_REVERT, params.into(), Zero::zero());
+                let _ =
+                    sdk::send::send(&our_addr, EMIT_SUBCALLS_REVERT, params.into(), Zero::zero())
+                        .ok();
```

Comment:
You don't need the `ok()` (but I'm not sure why you're not just calling `unwrap` to assert that this worked.

Reply:
The Result here is deliberately unused, and clippy was complaining. One of these sends is going to fail (see below), and ignoring that failure is part of the test scenario. I can document it.

Reply:
Why clippy, WHY!!!